### PR TITLE
Adding VSCode debug configurations

### PR DESCRIPTION
**Pull request by evqna**
After struggling for a while with setting up proper debugging for the app, I thought it would be useful to share my setup.
The outputCapture option is necessary to catch the logging output from Bunyan in the internal debugging console, which makes it easier to spot potential pitfalls not covered by our test assertions.

Signed-off-by: Antoine Salon <asalon@vmware.com>

### CODE REVIEW

**Comment by MVrachev** [comment]
I think this can be useful for future contributors.

**Comment by MVrachev** [comment]
But maybe wait for Erics opinion.

File: docs/localdev.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<!--
+    Copyright 2018 VMware, Inc.
```
Comment (ericwb) [inline docs/localdev.md]:
```suggestion
    Copyright 2019 VMware, Inc.
```

Comment (evqna) [reply docs/localdev.md]: Looks like I still need to get used to it!

File: .vscode/launch.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+            "type": "node",
+            "request": "launch",
+            "name": "Probot debug",
+            "program": "${workspaceFolder}/node_modules/probot/bin/probot-run.js",
+            "args": [
+                "./index.js"
+            ]
+        }
+    ]
+}
```
Comment (ericwb) [inline .vscode/launch.json]: sorry, another nit.  Can you add new line

**Pull request merged**